### PR TITLE
Initial commit for Babelfish v2.11

### PR DESCRIPTION
**Pull request by rishabhtanwar29**
### Description
* Add version info and upgrade test for 14.13.
* Add upgrade schedule file for 14.13.
* Update T-SQL extension version to 2.11.0 and add
babelfishpg_tsql--2.10.0--2.11.0.sql upgrade script.

Signed-off-by: Rishabh Tanwar <ritanwar@amazon.com>

### Test Scenarios Covered ###
* **Use case based -**

* **Boundary conditions -**

* **Arbitrary inputs -**

* **Negative test cases -**

* **Minor version upgrade tests -**

* **Major version upgrade tests -**

* **Performance tests -**

* **Tooling impact -**

* **Client tests -**

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: .github/configuration/upgrade-test-configuration.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ upgrade-version: [{
 {
   upgrade-path: [
     {
-      version: '14.12',
+      version: '14.13',
```
Comment (sumitj824) [inline .github/configuration/upgrade-test-configuration.yml:64]: Shouldn't we add new path instead of overiding previous one?

File: .gitignore [hunk 2]
```diff
@@ -22,6 +22,8 @@ contrib/babelfishpg_tsql/antlr/cmake_install.cmake
 contrib/babelfishpg_tsql/pl_reserved_kwlist_d.h
 contrib/babelfishpg_tsql/pl_unreserved_kwlist_d.h
 contrib/babelfishpg_tsql/sql/babelfishpg_tsql--[0-9].[0-9].[0-9]--[0-9].[0-9].[0-9].sql
+contrib/babelfishpg_tsql/sql/babelfishpg_tsql--[0-9].[0-9].[0-9]--[0-9].[0-9][0-9].[0-9].sql
+contrib/babelfishpg_tsql/sql/babelfishpg_tsql--[0-9].[0-9][0-9].[0-9]--[0-9].[0-9][0-9].[0-9].sql
```
Comment (sumitj824) [inline .gitignore]: Thanks for adding these pattern in ignore files.
Non-blocking: Can't we have something genric like digit*.digit*.digit*, although this is also fine.

**Review by sumitj824 — COMMENTED** [review]
LGTM

File: .github/configuration/upgrade-test-configuration.yml (hunk 1 above)
Comment (rishabhtanwar29) [reply .github/configuration/upgrade-test-configuration.yml:64]: We don’t want too many github actions unnecessarily, we already have 14.11 -> latest so updating 14.12 -> latest to 14.13 -> latest should be sufficient.

File: .gitignore (hunk 2 above)
Comment (rishabhtanwar29) [reply .gitignore]: Thanks for the suggestion, updated the expression.

**Review by sumitj824 — APPROVED** [review]

**Comment by rishabhtanwar29** [comment]
Also, added one missing skip pattern for normal sql script in .gitignore.

**Review by shardgupta — APPROVED** [review]

**Pull request merged**